unstructured/documents/elements.py

```diff
@@ -443,6 +443,14 @@ def apply(self, *cleaners: Callable):
         self.text = cleaned_text
 
 
+class Section(Text):
+    """A section of text consisting of a combination of elements."""
+
+    category = "Section"
```

Comment:
it feels odd that this is just another Element type since all the ones we have so far (and planned) are "granular." that being said, i guess it has to derive from an Element to be immediately transformable by cleaning or staging bricks.

what do you think of the name `CompositeElement` instead of `Section`?

PS, i think eventually (soon, even) there could a metadata field that is the base64 gzipped data of the original elements in this element, so the transformation to whatever we call this type could be reversible, plus retaining the full granular element data could still be useful for end user applications. not a blocker for this PR, of course.

Reply:
also, thank you for getting the ball rolling here!

Reply:
Just updated to `CompositeElement`. I like that better than `Section` since `section` is already a metadata field. And yeah reasoning was to make the outputs immediately consumable by the functions we already have that expect `Element` objects. Zipping the original elements and maintaining that also sounds great!